docs/source/development/style-guide/review-standards.rst

```diff
@@ -0,0 +1,65 @@
+.. _pull-request-review-standards:
+
+Pull Request and Review Standards
+---------------------------------
+
+Before any code is merged into our code base, it will need to be put up for pull request (PR) and reviewed. The pull request should be created following
```

Comment:
```suggestion
Before any code is merged into the code base, it will need to be put up for pull request (PR) and reviewed. The pull request should be created following
```

Reply:
@maxinelasp I think you missed some of these nits, or did you not want to accept them? I do sort of think we should avoid looking like dictating that this is "ours" and just go the generic route and talk about abstract "repository/organization" instead of people "us/our"

Reply:
Yes, apparently these commits got erased somehow during my last edit. Let me try and fix that